### PR TITLE
✨ 💄 Adding Favicons based on Laravel.io speech box

### PR DESCRIPTION
**Pull request by svpernova09**
Not sure happy with how this came out, but without a proper square logo it may be hard to find something to work well at that small size. Likely looks better on retina devices/mobile than in a desktop browser tab.

Potential solution for #10

### CODE REVIEW

**Comment by driesvints** [comment]
Heya! This is a good idea. Until we have a new logo this will serve fine. 

But I think we can improve one thing. Atm. the image you sent in will indeed look a little quirky in a browser tab because it's rectangle. In the old forum we did have a favicon which was square. It's still cached on my side so I managed to take a screenshot of it:

screen shot 2017-06-20 at 4 20 50 pm

Do you think you can perhaps update the favicons you sent in to have them look like the square version from the screenshot?

File: public/images/favicons/manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+ "name": "Laravel IO Pastbin",
```
Comment (driesvints) [inline public/images/favicons/manifest.json]: Rename "Laravel IO" to "Laravel.io"

Comment (driesvints) [inline public/images/favicons/manifest.json]: Rename "Pastbin" to "Pastebin"

File: resources/views/_fav_icons.blade.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<link rel="apple-touch-icon" sizes="57x57" href="/images/favicons/apple-icon-57x57.png">
```
Comment (driesvints) [inline resources/views/_fav_icons.blade.php]: Rename this file to `_favicons.blade.php`.

File: public/images/favicons/manifest.json (hunk 1 above)
Comment (svpernova09) [inline public/images/favicons/manifest.json]: haha, I'm good at typing :D

**Comment by svpernova09** [comment]
Code changes complete, and pushed new versions of a zoomed "IO" logo.

**Comment by driesvints** [comment]
Awesome work man! If you're up for it, you can send the exact same changes to the portal repo. If not, no worries, I'll open up an issue or do the changes myself :)

**Comment by svpernova09** [comment]
Will do

**Pull request merged**